### PR TITLE
ATLAS-136: Add support for PATCH in Atlas server

### PR DESCRIPTION
**Pull request by HelioStrike**
Updated the update markers route to use PATCH instead of POST.

### CODE REVIEW

**Comment by cintiadr** [comment]
I'm lost on this, I assume that's the api agreed with Burke

**Comment by HelioStrike** [comment]
@cintiadr Yup, PATCH is recommended when we're updating a resource, and Burke suggested that we replace POST with PATCH to update markers.

File: public/js/user.js [hunk 1]
```diff
@@ -269,17 +269,18 @@ function saveMarker(e) {
...
     var json = JSON.stringify(site);
+    var isUpdateRequest = isValidMarkerId(id);
     $.ajax({
-      //If marker id is a uuid, its an update request to '/marker/[id]'
-      //else it is an add request to '/marker'
-      url: "/marker/"+(isValidMarkerId(id)?id:""),
-      type: "POST",
+      //If marker id is a uuid, its an PATCH update request to '/marker/[id]'
+      //else it is an POST create request to '/marker'
+      url: "/marker/"+(isUpdateRequest?id:""),
```
Comment (bmamlin) [inline public/js/user.js]: Don't send a trailing slash when you create a marker:
```node
url: "/marker" + (isUpdateRequest ? "/"+id : ""),
```

Comment (HelioStrike) [reply public/js/user.js]: Done! :)

**Comment by cintiadr** [comment]
That's missing to merge this one?

**Pull request merged**